services/ec2/src/main/java/org/finra/gatekeeper/services/accessrequest/delegates/RevokeAccessServiceTask.java

```diff
@@ -131,6 +132,7 @@ public void execute(DelegateExecution execution) throws Exception{
                 emailServiceWrapper.notifyAdminsOfFailure(accessRequest, e);
                 logger.error("Error pushing to SNS topic upon request expiration. Request ID: " + accessRequest.getId());
             }
+            RequestEventLogger.logEventToJson(org.finra.gatekeeper.common.services.eventlogging.EventType.AccessExpired, accessRequest);
```

Comment:
put this at the end of the method call

Reply:
Didn't pull origin, that should be removed from the pr now